packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -291,6 +294,25 @@ ${bodyDetail}`
       if (context.payload.pull_request === undefined) {
         return;
       }
+      if (context.payload.action === 'labeled') {
+        // Only proceeds if `snippet-bot:force-run` label is added.
+        if (context.payload.pull_request.labels === undefined) {
+          return;
+        }
+        const labelFound = context.payload.pull_request.labels.some(
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (label: any) => {
```

Comment:
We should avoid using any here, we can make an interface like 
```
Label {
     name: string
    }
```

Reply:
Yeah, I can do that, but then we will have more complicated code than before ;)

Reply:
done